scripts/setup_imdb.py

```diff
@@ -37,7 +37,7 @@ def is_setup():
 
 # This file contains the IMDB dataset and is based on the specifications of the
 # original JOB repository: https://github.com/gregrahn/join-order-benchmark
-LOCATION = "https://bit.ly/hyriseimdbdataset"
+LOCATION = "https://github.com/hyrise/hyrise/releases/download/imdb_data_set_v1.01/imdb_data.zip"
 FILE_NAME = "imdb_data.zip"
```

Comment:
I am currently in contact with Matthias to find an HPI-internal solution. But it probably won't make it into this PR.